tests/func/test_repro.py

```diff
@@ -1465,9 +1477,11 @@ def repro_dir(tmp_dir, dvc, run_copy):
     stages["dir_file_copy"] = stage
 
     last_stage = tmp_dir / "dir" / DVC_FILE
+    deps = [os.fspath(origin_copy_2), os.fspath(dir_file_copy)]
     stage = dvc.run(
+        cmd="echo {}".format(" ".join(deps)),
```

Comment:
For these kind of stage that does not generate any outs, I have used `echo` or `ls` as a `cmd`.